android/build.gradle

```diff
@@ -3,7 +3,7 @@
 buildscript {
     ext {
         buildToolsVersion = "33.0.0"
-        minSdkVersion = 21
+        minSdkVersion = 26
```

Comment:
Woah, this is a problem. We'd be blocking 6.8% of users from using the app... That doesn't seem worth it for an improved camera feature. 

What specifically requires that we use a platform min of Android 8?

<img width="556" alt="254630961-b427f24b-c3cb-4a99-b160-e43021cfd0f5" src="https://github.com/Expensify/App/assets/10736861/c522158c-0222-4397-9db2-592a889fab0c">

Reply:
I see the following listed as high-level new features with Android SDK 26: 

<img width="526" alt="Screenshot 2023-10-25 at 09 19 12" src="https://github.com/Expensify/App/assets/10736861/c01dc3c8-a757-4edf-b2f3-6600401f1304">